lib/ex_unit/test/runner_test.exs

```diff
@@ -0,0 +1,23 @@
+Code.require_file "test_helper.exs", __DIR__
+
+defmodule ExUnitTest do
+  use ExUnit.Case
```

Comment:
Actually, nevermind, we are using the `ExUnit.configuration` and other process could change it.